pulumi/infra/lambda_.py

```diff
@@ -178,17 +179,17 @@ def __init__(
                 opts=import_aware_opts(f"{lambda_name}/live", parent=self),
             )
 
-        if forwarder:
-
-            # Lambda function output is automatically sent to log
-            # groups named like this; we create one explicitly for
-            # integration with the metric forwarder.
-            self.log_group = aws.cloudwatch.LogGroup(
-                f"{name}-log-group",
-                name=f"/aws/lambda/{lambda_name}",
-                opts=pulumi.ResourceOptions(parent=self),
-            )
+        # Lambda function output is automatically sent to log
+        # groups named like this; we create one explicitly so we can
+        # specify retention rules.
+        self.log_group = aws.cloudwatch.LogGroup(
+            f"{name}-log-group",
+            name=f"/aws/lambda/{lambda_name}",
```

Comment:
Would it make sense to do `/aws/lambda/deployname`? AWS APIs often take prefixes, so it would make some querying easier. NBD

Reply:
I believe that AWS automatically creates these log groups with this naming pattern. We're only creating it because we need a handle to pass it to the forwarder.